bubblewrap.c

```diff
@@ -1313,6 +1323,20 @@ main (int argc,
     clone_flags |= CLONE_NEWIPC;
   if (opt_unshare_uts)
     clone_flags |= CLONE_NEWUTS;
+  if (opt_unshare_cgroup) {
+    if (stat ("/proc/self/ns/cgroup", &sbuf))
+      {
+        if (errno == ENOENT)
+          {
+            die ("Cannot create new cgroup namespace because the kernel does not support it");
+          }
+        else
+          {
+            die_with_error ("stat on /proc/self/ns/cgroup failed");
+          }
+      }
```

Comment:
Coding style is to not have brackets around single-line if/else branches (unless one of the branches need it, then both should have it).